gson/src/test/java/com/google/gson/functional/ReflectionAccessTest.java

```diff
@@ -119,6 +119,7 @@ public Object read(JsonReader in) throws IOException {
    * <p>See https://github.com/google/gson/issues/1875
    */
   @Test
+  @SuppressWarnings("CatchFail")
```

Comment:
Maybe this should actually be fixed; unfortunately JUnit 4 has no `fail` method with `Throwable` parameter, but you could manually throw an `AssertionError` with cause:
```diff
- fail("Unexpected exception; test has to be run with `--illegal-access=deny`");
+ throw new AssertionError("Unexpected exception; test has to be run with `--illegal-access=deny`", e);
```

Reply:
I was thinking the same thing, incidentally.